app/models/like.rb

```diff
@@ -1,2 +1,8 @@
 class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post, class_name: 'Post', foreign_key: 'post_id'
+
+  def update_like_counter
+    post.increment!(:likes_counter)
+  end
```

Comment:
- [ ] Please call the method `update_like_counter` after save, It will update the `likes_counter` for the post when a like is created.
  - You can consider the below code.
``` ruby
after_save :update_like_counter